detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/SolutionInspector.cs

```diff
@@ -211,17 +263,26 @@ private List<ProjectFile> FindProjectFilesFromSolutionFile(string solutionPath,
             {
                 List<string> contents = new List<string>(File.ReadAllLines(solutionPath));
                 var projectLines = contents.FindAll(text => text.StartsWith("Project("));
-                foreach (string projectText in projectLines)
+                var projectDirectoryPackagesLines = contents.FindAll(text => text.Contains("Directory.Packages.props"));
+                foreach (string projectText in projectDirectoryPackagesLines)
                 {
-                    ProjectFile file = ProjectFile.Parse(projectText);
-                    if (file != null)
+                    if (!projectText.Equals("\t\tDirectory.Packages.props = Directory.Packages.props"))
```

Comment:
The tabs in this check look a bit suspicious. Is indentation always the same?

Reply:
Yes they are always the same.